src/Controls/src/Core/EnumerableExtensions.cs

```diff
@@ -103,13 +95,5 @@ public static int IndexOf<T>(this IEnumerable<T> enumerable, Func<T, bool> predi
 
 			return -1;
 		}
-
```

Comment:
Did you just remove this because we don't use it anywhere?

Reply:
It is now part of the .net SDK and was giving an ambiguous thing.